test_1_1/task_2/main.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+#include "bogosort.h"
+
+void randomizeArray(int *array, int arrayLength, int minValue, int maxValue) {
+    for (int i = 0; i < arrayLength; ++i) {
+        array[i] = minValue + (rand() % (maxValue - minValue));
+    }
+}
+
+void printArray(const int *array, int length) {
+    for (int i = 0; i < length; ++i) {
+        printf("%d", array[i]);
+        if (i != length - 1) {
+            printf(", ");
+        }
+    }
+    printf("\n");
+}
+
+int readValue(const char *prompt, const char *incorrectValueMessage) {
+    int value;
```

Comment:
```suggestion
    int value = 0;
```
Инициализируйте всё и всегда, если нет веских причин этого не делать